modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/java/com/liferay/dynamic/data/mapping/form/builder/internal/servlet/DDMFormFunctionsServlet.java

```diff
@@ -64,58 +64,53 @@ protected void doGet(
 			HttpServletResponse httpServletResponse)
 		throws IOException, ServletException {
 
-		Map<String, DDMExpressionFunction> ddmExpressionFunctions = null;
+		Map<String, DDMExpressionFunctionFactory> ddmExpressionFunctions = null;
 
-		try {
-			ddmExpressionFunctions = getDDMExpressionFunctions();
+		ddmExpressionFunctions = getDDMExpressionFunctions();
 
-			String languageId = ParamUtil.getString(
-				httpServletRequest, "languageId");
+		String languageId = ParamUtil.getString(
+			httpServletRequest, "languageId");
 
-			JSONArray jsonArray = toJSONArray(
-				ddmExpressionFunctions.entrySet(),
-				LocaleUtil.fromLanguageId(languageId));
+		JSONArray jsonArray = toJSONArray(
+			ddmExpressionFunctions.entrySet(),
+			LocaleUtil.fromLanguageId(languageId));
 
-			httpServletResponse.setContentType(ContentTypes.APPLICATION_JSON);
-			httpServletResponse.setStatus(HttpServletResponse.SC_OK);
+		httpServletResponse.setContentType(ContentTypes.APPLICATION_JSON);
+		httpServletResponse.setStatus(HttpServletResponse.SC_OK);
 
-			ServletResponseUtil.write(
-				httpServletResponse, jsonArray.toJSONString());
-		}
-		finally {
-			if (ddmExpressionFunctions != null) {
-				_ddmExpressionFunctionTracker.ungetDDMExpressionFunctions(
-					ddmExpressionFunctions);
-			}
-		}
+		ServletResponseUtil.write(
+			httpServletResponse, jsonArray.toJSONString());
 	}
 
-	protected Map<String, DDMExpressionFunction> getDDMExpressionFunctions() {
+	protected Map<String, DDMExpressionFunctionFactory>
+		getDDMExpressionFunctions() {
```

Comment:
Since you're returning a Map of  `DDMExpressionsFunctionFactory` the method should be named accordingly  to something like `getDDMExpressionFunctionFactories`